tests/mocha/navigation_test.js

```diff
@@ -32,10 +130,12 @@ suite('Navigation', function() {
       chai.assert.equal(Blockly.Navigation.currentState_,
           Blockly.Navigation.STATE_WS);
     });
-
+    // More tests:
+    // - nested categories
     teardown(function() {
```

Comment:
Can you move the teardown to just below setup so it's easier to verify they're paired?